src/magicbox/SuggestionsManager.ts

```diff
@@ -471,21 +504,21 @@ export class SuggestionsManager {
     return null;
   }
 
-  private addSelectedStatus(suggestion: HTMLElement): void {
+  private addSelectedStatus(element: HTMLElement): void {
     const selected = this.element.getElementsByClassName(this.options.selectedClass);
     for (let i = 0; i < selected.length; i++) {
       const elem = <HTMLElement>selected.item(i);
       this.removeSelectedStatus(elem);
     }
-    $$(suggestion).addClass(this.options.selectedClass);
-    this.updateSelectedSuggestion(suggestion.innerText);
-    this.updateAreaSelectedIfDefined(suggestion, 'true');
+    $$(element).addClass(this.options.selectedClass);
+    this.updateAreaSelectedIfDefined(element, 'true');
   }
 
-  private updateSelectedSuggestion(suggestion: string) {
+  private updateSelectedSuggestion(suggestion: HTMLElement) {
     $$(this.root).trigger(OmniboxEvents.querySuggestGetFocus, <IQuerySuggestSelection>{
-      suggestion
+      suggestion: suggestion.innerText
     });
+    this.displaySearchResultPreviewsForSuggestion(suggestion);
```

Comment:
This is the starting point where search result previews are queried then displayed.